### PR TITLE
APPUiO - Swiss Container Platform

### PR DESCRIPTION
**Pull request by phil-pona**

### CODE REVIEW

File: profiles/appuio.json [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+    ]
+  },
+  "extensible": true,
+  "extensible": true,
+  "infrastructures": [
+    {
+      "continent": "EU",
+      "country": "CH",
+      "region": "",
```
Comment (stefan-kolb) [inline profiles/appuio.json]: Can you please add the region/city of your servers please.

File: profiles/appuio.json [hunk 2]
```diff
@@ -0,0 +1,221 @@
+{
+  "name": "APPUiO - Swiss Container Platform",
```
Comment (stefan-kolb) [inline profiles/appuio.json]: Just use APPUiO here. otherwise the filename must be different.

File: profiles/appuio.json [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  "revision": "2016-10-05",
+  "url": "https://appuio.ch",
+  "vendor_verified": "2016-10-05",
+  "status": "production",
+  "status_since": "2016-10-05",
+  "type": "Generic",
+  "hosting": {
+    "public": true,
+    "private": true
```
Comment (stefan-kolb) [inline profiles/appuio.json]: You can also add `vps: true` as you offer to host private single tenant instances for your customers?!

File: profiles/appuio.json [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  },
+  "pricing": [
+    {
+      "model": "fixed"
```
Comment (stefan-kolb) [inline profiles/appuio.json]: missing `"period": "monthly"`

**Comment by stefan-kolb** [comment]
Thank you for your contribution :smile: !

**Comment by phil-pona** [comment]
Thank you for approving ;-)

**Pull request merged**